package.json

```diff
@@ -19,7 +19,9 @@
     "jsii-pacmak": "^0.21.1",
     "jsii-rosetta": "^0.21.1",
     "lerna": "^3.20.2",
-    "typescript": "~3.7.5"
+    "typescript": "~3.7.5",    
+    "github-api": "^3.3.0",
+    "make-runnable": "^1.3.6"
```

Comment:
It feels odd that these are added to the root of the repo. Maybe we can put this project under `tools/`?

Reply:
I agree. My thought was that this would eventually evolve to a command line tool that goes under `tools`. Do you mean that we can just put this code under tools even now? Without creating a proper command line?

Reply:
Yap

Reply:
Will do 👍